housingbayarea/bloom
# fix: remove application due date question

# Pull Request Template

## Issue Overview

This PR addresses #600 


## Description

This PR removes `Is there an application due date?` from listing create process, and makes `application due date/time` editable also for `Open Waitlist`.

## How Can This Be Tested/Reviewed?

(2 example videos in task for reference)
create listing -> save draft with `What is the listing availability?` on `open waitlist` -> after edit it should be possible to edit `application due date/time`

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
